client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.tsx

```diff
@@ -103,8 +104,6 @@ export default function WPCheckoutOrderReview( {
 		[ reduxDispatch ]
 	);
 
-	const selectedSiteData = useSelector( getSelectedSite );
```

Comment:
I'm a little worried about this. I lifted this fetch request up to the WPCheckout component so I could pass it down to both the checkout line items and the checkout details summary in the sidebar, both are children of WPCheckout. Since I am passing it back to `wp-checkout-order-review` the original usage of `selectedSiteData` is still receiving the same data, just from props rather than fetching the data from within the `WPCheckoutOrderReview` component directly.